GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -901,6 +964,85 @@ namespace gdjs {
       }
     }
 
+    getNetworkSyncData(): SoundManagerSyncData {
+      const freeMusicsDatas: SoundSyncData[] = [];
+      for (const sound of Object.values(this._freeMusics)) {
+        freeMusicsDatas.push(sound.getNetworkSyncData());
+      }
+      const freeSoundsDatas: SoundSyncData[] = [];
+      for (const sound of Object.values(this._freeSounds)) {
+        freeSoundsDatas.push(sound.getNetworkSyncData());
+      }
+      const musicsDatas: SoundSyncData[] = [];
+      for (const sound of Object.values(this._musics)) {
+        musicsDatas.push(sound.getNetworkSyncData());
+      }
+      const soundsDatas: SoundSyncData[] = [];
+      for (const sound of Object.values(this._sounds)) {
+        soundsDatas.push(sound.getNetworkSyncData());
+      }
+      return {
+        globalVolume: this._globalVolume,
+        availableResources: this._availableResources,
+        cachedSpatialPosition: this._cachedSpatialPosition,
```

Comment:
here you're passing availableResources & cachedSpatialPosition but you're not using them when updating the network sync data?